frontend/src/components/individual-recipe/Tutorial.js

```diff
@@ -122,7 +122,15 @@ class Tutorial extends Component {
   }
 
   finishCooking() {
```

Comment:
```suggestion
  finishCooking(recipe) {
```

Reply:
It's because once you click on the button that redirects to another page, `this` doesn't refer to Tutorial.js, it will refer to something undefined. That's why you need to accept the recipe as a parameter.